app/src/main/java/com/nextcloud/talk/adapters/items/UserItem.java

```diff
@@ -62,20 +52,18 @@
 public class UserItem extends AbstractFlexibleItem<UserItem.UserItemViewHolder> implements
     ISectionable<UserItem.UserItemViewHolder, GenericTextHeaderItem>, IFilterable<String> {
 
-    private static final float STATUS_SIZE_IN_DP = 9f;
-    private static final String NO_ICON = "";
+    public static final String PARTICIPANT_SOURCE_CIRCLES = "circles";
+    public static final String PARTICIPANT_SOURCE_GROUPS = "groups";
+    public static final String PARTICIPANT_SOURCE_USERS = "users";
 
-    private Context context;
-    private Participant participant;
-    private UserEntity userEntity;
+    private final Participant participant;
+    private final UserEntity userEntity;
     private GenericTextHeaderItem header;
     public boolean isOnline = true;
 
-    public UserItem(Context activityContext,
-                    Participant participant,
+    public UserItem(Participant participant,
```

Comment:
maybe we should rename UserItem to ContactItem (according to R.layout.rv_item_contact and ContactsController)